layers/gd/Dockerfile

```diff
@@ -0,0 +1,55 @@
+ARG PHP_VERSION
+FROM bref/build-php-$PHP_VERSION as ext
+
+# Install required build libraries
+RUN set -ue \
+    ; LD_LIBRARY_PATH= \
+    ; yum -y install \
+        libwebp-devel \
+        libXpm-devel \
+        libpng-devel \
+        libjpeg-devel \
+        freetype-devel \
+    ; yum clean all \
```

Comment:
@drAlberT I'm not sure this is usefull?

Reply:
as we copy needed files away from the base image it should be not influent in fact. It is a best practice to reduce layers size on build... 

should be pruneable

Reply:
Yes but as we are multi stage we don't care here